binplace.targets

```diff
@@ -35,6 +35,9 @@
       <TargetingPackDir Include="$(RefPath)" />
       <TargetingPackDir Condition="'$(IsNETCoreAppRef)' == 'true'" Include="$(BinDir)netcoreapp\pkg\ref" />
     </ItemGroup>
+    <ItemGroup Condition="'$(NuGetTargetMoniker)' == '' Or $(NuGetTargetMoniker.Contains('UAP'))">
```

Comment:
I'm not sure I follow. It will be the same assembly just loaded from a different ref directory. The entire idea is that we want an easy reference resolution from flat targeting pack for each vertical BuildConfiguration and not have to try and add a bunch of logic to pick the correct reference for every library. We aren't in the contract store or nuget resolution world when we build a vertical. To resolve a Reference you are only looking in one targeting pack directory.